README.adoc

```diff
@@ -95,3 +95,12 @@ The server will stay up and self-update automatically.
 You can publish the site in your cloned repo to see the result of your changes.
 
 First, enable GitHub pages in your forked repository and set it to publish to the gh-pages branch.
+
+== Generating a new Edition
+
+TBD
+
+[Important]
+====
+Remember to update the open graph values located at the `_layouts/default.html` page, they are used for the thumbnail generation at every social network.
```

Comment:
Do we need to change anything in there? Probably everything should be changed in the config file (`homepage.json`), isn't it?

Reply:
yes, we need to change the reference image. In the `homepage.json` it will be the same path.

Reply:
Ah, now I understood! Then the one we are using nowadays is the one ... from 2019? 😅https://github.com/barcelonajug/jbcnconf_web/blob/master/2021/assets/img/logos/og-image.png

Should we instead use for example the image you designed for Hopin?

Reply:
Yes, that's right. I was thinking of using this squared one because it's more suitable for social networks, but I can replace it with the large one.